src/components/Map/Map.js

```diff
@@ -67,6 +70,13 @@ class Map extends PureComponent {
     this.onPointerMoveRef = map.on('pointermove', e => this.onPointerMove(e));
   }
 
+  componentDidUpdate(prevProps) {
+    const { highlight } = this.props;
+    if (prevProps.highlight !== highlight) {
+      console.log('TODO: highlight!!!', highlight);
```

Comment:
Pronlem is how do we know which highlight to apply. Highlight from a layer, from the search , from the app...

Reply:
`highlight` would contain an OpenLayers feature which will be rendered in a VectorLayer with a generic highlight style. If our highlighting requirements are more complex we should propably implement this directly in the Search component and SearchService.